docs(gen): docs content review (#2759)
scaleway/docs-content: dedibox/kvm-over-ip/how-to/supermicro.mdx

  - Your Dedibox server has a [Supermicro](https://www.supermicro.com/en/solutions/management-software/bmc-resources) KVM-over-IP device
</Message>

## Creating a KVM Session

1. Access your [Dedibox console](https://console.online.net/) and navigate to **Server** > **Server List**. This will display a list of your dedicated servers.
2. Select the server you wish to access by clicking its name. This will open the server's status page.
3. Click **KVM OVER IP** on the right-hand menu. A pop-up window will appear.
    <Lightbox src="scaleway_KVM.webp" alt="" />
4. Confirm your acceptance of the terms for KVM access by clicking **I Accept**.

> 5. Provide the authorized IPv4 address for the KVM connection. The form will auto-fill with the IP address of your current internet connection. Then click **Create** to generate your credentials.

Reviewer: ```suggestion
5. Provide the authorized IPv4 address for the KVM connection. The form will autofill with the IP address of your current internet connection. Then click **Create** to generate your credentials.
```

    <Message type="note">
      Currently, only IPv4 addresses are accepted.
    </Message>

Once completed, the connection URL and your credentials will be displayed. Click on the provided link to access the KVM interface.

## Accessing the KVM-over-IP device

1. Open the connection URL of your KVM in your web browser and log in using the credentials provided during access creation.
2. In the KVM menu, click on **Virtual Media** > **CD-ROM Image**:
    <Lightbox src="scaleway_supermicro_vm.webp" alt="" />
3. Enter the image file path of your OS image:
    <Lightbox src="scaleway_supermicro_nas.webp" alt="" />

    <Message type="tip">
    You can find the image file path for your OS on our [NAS](http://virtualmedia.online.net). <br />

>     If your preferred OS is not available on the NAS, please contact our support team for assistance.

Reviewer: ```suggestion
    If your preferred OS is not available on the NAS, contact our support team for assistance.
```

Your server will boot into installation mode. Follow the operating system's installation wizard to complete the installation.

<Lightbox src="scaleway_supermicro_install.webp" alt="" />

## Installing Windows network drivers

Microsoft Windows Server does not support the network adapter used in Supermicro servers by default. Follow these steps to manually install network drivers using the KVM-over-IP device:

1. [Create](#creating-a-kvm-session) a Supermicro KVM session.
2. Open the connection URL of the KVM in your web browser and log in using the provided credentials.
3. [Download](https://www.supermicro.com/wdl/CDR_Images/CDR-A1-A2-UP/CDR-A1-A2-UP_1.02_for_Intel_A1_A2_UP_platform.iso.iso) the driver package from Supermicro to your local computer.
    <Message type="tip">
      You can also download a local copy of the file directly onto your server using Scaleway's NAS: [https://virtualmedia.online.net/supermicro/CDR-A1-UP_1.00_for_Intel_A1_UP_platform.iso](https://virtualmedia.online.net/supermicro/CDR-A1-UP_1.00_for_Intel_A1_UP_platform.iso)
    </Message>
4. Access the KVM-over-IP device by clicking **Remote Console Preview**.

> 5. Click **Virtual Media** > **Virtual Storage**. A popup will appear:

Reviewer: ```suggestion
5. Click **Virtual Media** > **Virtual Storage**. A pop-up will appear:
```
